Polar/README.md

```diff
@@ -1,3 +1,73 @@
 # Polar Datacube Generation
 
-Information dedicated to the generation of the Polar Datacube.
\ No newline at end of file
+Information dedicated to the generation of the Polar Datacube.
+
+## Cube generation process
+
+The cube generation process is divided in four phases:
+
+### 1. Downloading required raw datasets
```

Comment:
Please add a note, that new directories are created and the input data downloaded there (best to mention where they are created) otherwise it is a bit surprising, the data is downloaded into a subdir outside of the polar directory which is the current working dir.